modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -7,15 +7,20 @@
 [id="cnf-provisioning-real-time-and-low-latency-workloads_{context}"]
 = Provisioning real-time and low latency workloads
 
-Many industries and organizations need extremely high performance computing and might require low and predictable latency, especially in the financial and telecommunications industries. For these industries, with their unique requirements, {product-title} provides a Performance Addon Operator to implement automatic tuning to achieve low latency performance and consistent response time for {product-title} applications.
+Many industries and organizations need extremely high performance computing and might require low and predictable latency, especially in the financial and telecommunications industries. For these industries, with their unique requirements, {product-title} provides a Performance Addon controller to implement automatic tuning to achieve low latency performance and consistent response time for {product-title} applications.
```

Comment:
Do we need to update Performance Addon Operator -> Performance Addon controller throughout the docs? 
According to Atom (not sure which feature branch):  97 results found in 35 file for Performance Addon Operator

Reply:
Yes, that is covered by [TELCODOCS-580](https://issues.redhat.com//browse/TELCODOCS-580). There are several writers working on the relevant files and we're working together to catch all instances.